php_sql/sql_02/db_manipulation.php

```diff
@@ -0,0 +1,83 @@
+<?php
+$configs = include('configuration_properties.php');
+
+function getDatabaseConnection()
+{
+    global $configs;
+    $host = $configs['host'];
+    $dbname = $configs['database_name'];
+    $username = $configs['username'];
+    $password = $configs['password'];
+
+    $connection = new PDO("mysql:host=$host;dbname=$dbname", $username, $password);
+    return $connection;
+}
+
+function insertElective($title, $description, $lecturer)
+{
+    global $configs;
+    $table = $configs['table_name'];
+    $connection = getDatabaseConnection();
+    $insertQuery = "INSERT INTO $table(title,description,lecturer) VALUES (:title, :description, :lecturer)";
+
+    $preparedSql = $connection->prepare($insertQuery) or die("Failed to prepare the sql query.");
+    $preparedSql->bindParam(':title', $title);
+    $preparedSql->bindParam(':description', $description);
+    $preparedSql->bindParam(':lecturer', $lecturer);
+    $preparedSql->execute() or die("Failed to execute sql query.");
+}
+
+
+function getElective($id)
+{
+    if (!isExistingId($id)) {
+        die("The elective with the given id: $id is not existing.");
+    }
+    global $configs;
+    $table = $configs['table_name'];
+    $connection = getDatabaseConnection();
+    $selectQuery = "SELECT * FROM $table WHERE id = :id;";
+
+    $result = $connection->prepare($selectQuery) or die("Failed to prepare sql query.");
+    $result->bindParam(':id', $id);
+    $elective = $result->execute() or die("Failed to execute sql query.");
+    while ($row = $result->fetch(PDO::FETCH_ASSOC)) {
+        echo "Subject ID: " . $row["id"] . "<br/>Title: " . $row["title"] . "<br/>Description: " . $row["description"] . "<br/>Lecturer: " . $row["lecturer"] . "<br/><br/>";

```

Comment:
This could become a bit more readable if this is initialized on multiple lines with each line ending with the `<br/>` expression.